docs/genai/howto/install.md

```diff
@@ -13,40 +13,46 @@ nav_order: 1
 * TOC placeholder
 {:toc}
 
-## Python package release candidates
+## Python packages
 
 ```bash
 pip install numpy
-pip install onnxruntime-genai --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
+pip install onnxruntime-genai --pre
+```
+Append `-directml` for the library that is optimized for DirectML on Windows
+
+```bash
+pip install numpy
+pip install onnxruntime-genai-directml --pre
 ```
 
 Append `-cuda` for the library that is optimized for CUDA environments
 
 ```bash
+pip install numpy
 pip install onnxruntime-genai-cuda --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
```

Comment:
Should we change any references of `--index-url` to `--extra-index-url`? Some users run into [installation issues](https://github.com/microsoft/onnxruntime-genai/pull/448) without it.

Reply:
There is a security vulnerability with --extra-index-url. We had to change all of the instances previously